package.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "symbol-wallets-lib",
+    "version": "0.1.0",
+    "description": "Symbol Wallets Library",
+    "main": "build/index.js",
+    "scripts": {
+        "prepare": "npm run build",
+        "build": "tsc",
+        "lint": "eslint --cache src/ --ext .ts",
+        "lint:fix": "eslint src/ --ext .ts --fix"
+    },
+    "contributors": [
+        {
+            "name": "Peersyst Technology",
+            "email": "dev@peersyst.com"
+        },
+        {
+            "name": "NEM Software Ltd",
+            "email": "info@nem.software"
+        }
+    ],
```

Comment:
This is a new project, right?
If yes I don't think we need this.

Reply:
This is from `symbol-paper-wallets` project. Now it is part of `wallets-lib`